### DISCUSSION TITLE
Adding Testcases for each concepts with respect to all Languages

### DISCUSSION THREAD

**Discussion by Suryac72** [dev-madhurendra/OOPs#25] [Ideas]
Hi @dev-madhurendra , can we setup convention of adding test cases for each type of language dependency oops concepts??

**Comment by dev-madhurendra** [comment]
@Suryac72 Yeah, It is fine but first we have to add code along with a good explanation of each and every concepts so that anyone can understand by just reading and while reviewing the code.
